cpp/ps0002/Task2.cpp

```diff
@@ -4,13 +4,18 @@ using namespace std;
 
 int main()
 {
-    short int building_members = 15;
-    short int num_alternatives_to_medicines = 3;
-    int company_members = 125;
-    short int bakery_prices = 5;
-    char first_letter_in_the_name = 'A';
-    double num_of_births = 1784596;
-    float num div 7 = 1.3333;
-    double num_electricity_company_bills_in_egypt = 52379641;
-    unsigned int age_yuang_and_old_people = 4200000;
+    short int building_members = 15; // 0.75/0.75
+    short int num_alternatives_to_medicines = 3; // 0.75/0.75
+    short int company_members = 125; // 0.5/0.75
+    float bakery_prices = 5; // 0.5/0.75
+    char first_letter_in_the_name = 'A'; // 0.75/0.75
+    unsigned int num_of_births = 1784596; // 0.5/0.75
+    double num_div_7 = 1.3333; // 0.25/0.75
+    double num_electricity_company_bills_in_egypt = 52379641; // 0.75/0.75
+    unsigned int age_yuang_and_old_people = 4200000; // 0.75/0.75
+    // -0.25 mark cuz of large variable name
+    // total 5.25/6.75

```

Comment:
- [x] all variables required in the document are declared
- [ ] variables are declared with the correct data type
- [x] using correct literals and notations for assigning initial values to the variables